tests/run/pure_py.py

```diff
@@ -17,10 +17,12 @@ def test_sizeof():
     True
     True
     True
+    True
     """
     x = cython.declare(cython.bint)
     print(cython.sizeof(x) == cython.sizeof(cython.bint))
-    print(cython.sizeof(cython.char) <= cython.sizeof(cython.short) <= cython.sizeof(cython.int) <= cython.sizeof(cython.long) <= cython.sizeof(cython.longlong))
+    print(cython.sizeof(cython.char) <= cython.sizeof(cython.short) <= cython.sizeof(cython.int))
+    print(cython.sizeof(cython.int) <= cython.sizeof(cython.long) <= cython.sizeof(cython.longlong))
```

Comment:
I'd rather see `cython.sizeof` assigned to a variable `sizeof` here. Maybe not in all cases, since variation in usage is good for this test, but for this specific line, it seems like a good idea.